stdlib/REPL/test/repl.jl

```diff
@@ -1987,3 +1987,44 @@ end
     write(proj_file, "name = \"Bar\"\n")
     @test get_prompt("--project=$proj_file") == "(Bar) pkg> "
 end
+
+# Issue #58158 add alias for Char display in REPL
+@testset "REPL show_repl Char alias" begin
+    # Test character with a known emoji alias
+    buf = IOBuffer()
+    io = IOContext(buf, :color => true)
+    REPL.show_repl(io, MIME("text/plain"), '😼')
+    output = String(take!(buf))
```

Comment:
FYI, you can simplify this using `sprint`:
```suggestion
    output = sprint(REPL.show_repl, MIME("text/plain"), '😼'; context=(:color => true))
```

Reply:
Thanks for the great suggestion, @ararslan! I've updated the tests in `stdlib/REPL/test/repl.jl` to use `sprint` as you recommended. It definitely makes the code cleaner.

The changes have been pushed, and the local tests are still passing. Ready for another look when you have a moment!